docs/source/dataset_summary.rst

```diff
@@ -101,7 +100,10 @@ DOI: https://doi.org/10.1088/1741-2552/ac38cf
    :class: sortable
 
    :class:`Thielen2021`,30,8,20,5,31.5s,18900 NT / 18900 T,512Hz,1,Gold codes,60Hz
-
+   :class:`CastillosCVEP100`,12,32,4,0.25s,,512Hz,1,Burst-CVEP,60Hz
+   :class:`CastillosCVEP40`,12,32,4,0.25s,,512Hz,1,Burst-CVEP,60Hz
+   :class:`CastillosBurstVEP40`,12,32,4,0.25s,,512Hz,1,Burst-CVEP,60Hz
+   :class:`CastillosBurstVEP100`,12,32,4,0.25s,,512Hz,1,Burst-CVEP,60Hz
```

Comment:
`Trials length` is not consistent with the definition of Thielen2021.

I think we should have two columns for the c-vep datasets: `Trials length` and `Epochs length` see https://github.com/NeuroTechX/moabb/pull/557/files#r1559388876

Reply:
Yes It can be a good idea to have these both columns. But at the same time, we want to know how long are the windows of the data when getting the data of the dataset, no?

Reply:
Yes but in the future, the plan for c-VEP datasets is to both have the option to load the whole sequences or the individual flashes separately. So both columns will be relevant

Reply:
In fact, the epoch length is a hyper-parameter that anyone could change depending on their needs or wishes. Instead, the trial-length is a real constant, it cannot be changed, as the stimulus protocol as carried out during the experiment defines how long was stimulated, and thus how long a trial took.

Reply:
I didn't thin about the future, sorry. This is a good point so the trial length should be here 2.2s (time of the stimulation of the code) and not 0.25s if I understood correctly

Reply:
@sebVelut just to clarify: In the Castillos datasets, does the target character stay the same for 2.2s (one code presentation exactly) or do the character presentation and code have different durations?

Reply:
yes @PierreGtch, I updated the table